packages/webapp/src/store/api/apiSlice.ts

```diff
@@ -61,7 +61,56 @@ import type {
   SensorReadings,
   IrrigationPrescription,
 } from './types';
-import i18n from '../../locales/i18n';
+
+/**
+ * These tags have endpoints that do not return farm specific data, and are not created by a farm
+ *
+ *  LiteFarm provides these data as defaults
+ */
+export const LibraryTags = [
+  'AnimalSexes',
+  'AnimalIdentifierTypes',
+  'AnimalIdentifierColors',
+  'AnimalMovementPurposes',
+  'AnimalOrigins',
+  'AnimalUses',
+  'AnimalRemovalReasons',
+  'DefaultAnimalBreeds',
+  'SoilAmendmentMethods',
+  'SoilAmendmentPurposes',
+  'SoilAmendmentFertiliserTypes',
+];
+
+/**
+ * These tags contain endpoints that return farm specific data
+ *
+ * These data should not persist when switching farms,
+ * or should be stored in a separate farm store.
+ */
+export const FarmTags = [
+  'Animals',
+  'AnimalBatches',
+  'CustomAnimalBreeds',
+  'CustomAnimalTypes',
+  'FarmAddon',
+  'IrrigationPrescriptions',
+  'SoilAmendmentProduct',
+  'Sensors',
+  'SensorReadings',
+  'Weather',
+];
+
+/**
+ * These tags contain endpoints that could either return farm specific data
+ * or farm neutral defaults data.
+ *
+ * For data safety these data should also not persist when switching farms,
+ * or should be stored in a separate farm store.
+ */
+export const FarmLibraryTags = [
+  // 'count' param returns farm specific data
+  'DefaultAnimalTypes',
```

Comment:
This is an abnormal endpoint compared to our others. It might be worth retrieving count from a different endpoint.